doc/api/repl.markdown

```diff
@@ -290,9 +292,24 @@ the following values:
 You can use your own `eval` function if it has following signature:
 
     function eval(cmd, context, filename, callback) {
```

Comment:
this definitely needs to be rebased before landing to pick up the code block formatting changes